hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/TestDatanodeConfiguration.java

```diff
@@ -199,10 +200,11 @@ void useMinFreeSpaceIfBothMinFreeSpacePropertiesSet() {
     DatanodeConfiguration subject = conf.getObject(DatanodeConfiguration.class);
 
     assertEquals(minFreeSpace, subject.getMinFreeSpace());
-    assertEquals(DatanodeConfiguration.MIN_FREE_SPACE_UNSET, subject.getMinFreeSpaceRatio());
+    assertEquals(.5f, subject.getMinFreeSpaceRatio());
```

Comment:
Please change this unit test function name.

Reply:
updated